commerce_coordinator/apps/commercetools/tests/sub_messages/test_tasks.py

```diff
@@ -252,6 +254,18 @@ def setUp(self):
                     self.mock.update_return_payment_state_after_successful_refund
             }
         )
+        # TODO: Properly mock the Payment object.
```

Comment:
Do we still need this TODO?

Reply:
Yes, `get_payment_by_key` is mocked with `CTPaymentByKey` return value, and in this mocking transaction amount is set integer but its type should be CTMoney. By updating that, getting errors in other unit tests so we should handle mocking for this case in proper way.

Reply:
Okay. Please create a ticket in the backlog to fix this test in upcoming sprints.

Reply:
Ok, The tests are working fine and covering the code lines but we can improve the mocking approach.